tests/i18n/tests.py

```diff
@@ -296,6 +315,11 @@ def test_language_bidi(self):
         with translation.override(None):
             self.assertIs(get_language_bidi(), False)
 
+    def test_language_bidi_null(self):
+        self.assertIs(trans_null.get_language_bidi(), False)
+        with translation.override(None):
```

Comment:
This should use `with self.setttings(LANGUAGE_CODE=...)` to test the `True` case.

Reply:
Because the class  has no attribute `setttings`, I defined new test and used `@override_settings`